tests/migration_test.go

```diff
@@ -1571,9 +1572,32 @@ var _ = Describe("[rfe_id:393][crit:high[vendor:cnv-qe@redhat.com][level:system]
 					targetNode.Name,
 					targetNode.Name,
 				))
+
+				By("Checking that all migrated VMIs have the new pod IP address")
+				for _, vmi := range vmis {
+					Eventually(func() error {
+						newvmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+						Expect(err).ToNot(HaveOccurred())
+						vmiPod := tests.GetRunningPodByVirtualMachineInstance(newvmi, tests.NamespaceTestDefault)
+
+						for _, network := range newvmi.Spec.Networks {
+							if network.NetworkSource.Pod != nil {
+								for _, iface := range newvmi.Status.Interfaces {
+									if iface.Name == network.Name {
+										if iface.IP == vmiPod.Status.PodIP {
+											return nil
+										} else {
+											return goerrors.New(fmt.Sprintf("VMI IP:%s not equal to Pod IP:%s", iface.IP, vmiPod.Status.PodIP))
```

Comment:
You could use `fmt.Errorf`here instead of explicitly creating a new error... just a matter of readability IMO.

Reply:
Changed.